### PR TITLE
UI components for annotation service

### PR DESCRIPTION
**Pull request by yisehak-awm**
This PR includes the following changes:

1. #### Annotation service UI components ####
**AnnotationService** - The root component for the annotation service UI
**AnnotationSelection** - Component for selecting annotations.
**GeneSelection** - Component for specifying genes to be annotated.
**GOFilter** - Component for specifying filter values for gene-go annotation.
**GenePathwayFilter** - Component for specifying filter values for gene pathway annotation.
**AnnotationResultDownload** - Component that displays a button to download result as a scheme file.
**AnnotationResultVisualizer** - Component that uses a cytoscape graph to visualize result.
**visualizer.config.js** - configuration file for the cytoscape visualizer

2.A minor bug fix for an issue causing radio buttons to malfunction on the `moses-service` form.

### CODE REVIEW

File: src/components/service/ServiceMappings.js [hunk 1]
```diff
@@ -1,63 +1,107 @@
...
+    this.serviceOrgIDToComponent[
+      this.generateUniqueID("snet", "time-series-anomaly-discovery")
+    ] = TimeSeriesAnomalyDiscoveryService;
+    this.serviceOrgIDToComponent[
+      this.generateUniqueID("snet", "moses-service")
+    ] = MosesService;
+    this.serviceOrgIDToComponent[
+      this.generateUniqueID("snet", "annotation-service")
+    ] = AnnotationService;
+  }
```
Comment (ferrouswheel) [inline src/components/service/ServiceMappings.js:49]: Any reason for this format? A lot less easy to overview than it was...

**Comment by ferrouswheel** [comment]
If this is for gene annotation can you change the name to reflect that? "Annotation" is used in many contexts in machine learning. E.g. Video Annotation, NLP annotation, etc.

File: src/networks.js [hunk 2]
```diff
@@ -30,10 +30,10 @@ export const NETWORKS = {
...
 
-  export function getMarketplaceURL(chainId) {
-    return (chainId in NETWORKS ? NETWORKS[chainId]['marketplace'] : undefined);
-  }
-  
-  export function getProtobufjsURL(chainId) {
-    return (chainId in NETWORKS ? NETWORKS[chainId]['protobufjs'] : undefined);
-  }
\ No newline at end of file
+export function getMarketplaceURL(chainId) {
```
Comment (raamb) [inline src/networks.js:35]: Lets undo these changes. Dont want the DApp files being modified as part of the Service UI PR

File: src/components/service/ServiceMappings.js (hunk 1 above)
Comment (yisehak-awm) [reply src/components/service/ServiceMappings.js:49]: Prettier plugin for VS Code did that, will undo it.

**Comment by yisehak-awm** [comment]
Closing this PR to send a new one with no conflicts
